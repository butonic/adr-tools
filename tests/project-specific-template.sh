adr init adrs
mkdir -p adrs/templates
cat > adrs/templates/template.md <<EOF
---
title: "NUMBER. [short title of solved problem and solution]"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0005-example.md
custom: foo
---

* Status: [proposed | accepted | deprecated | superseded by [ADR-0005](0005-example.md) | â€¦] <!-- optional -->
* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->

## Context and Problem Statement

EOF
adr new Aaa Bbb
adr new Ccc Ddd
cat adrs/0002-aaa-bbb.md
cat adrs/0003-ccc-ddd.md
