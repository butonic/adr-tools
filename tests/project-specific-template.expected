adr init adrs
adrs/0001-use-markdown-architectural-decision-records.md
mkdir -p adrs/templates
cat > adrs/templates/template.md <<EOF
---
title: "NUMBER. [short title of solved problem and solution]"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0005-example.md
custom: foo
---

* Status: [proposed | accepted | deprecated | superseded by [ADR-0005](0005-example.md) | â€¦] <!-- optional -->
* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->

## Context and Problem Statement

EOF
adr new Aaa Bbb
adrs/0002-aaa-bbb.md
adr new Ccc Ddd
adrs/0003-ccc-ddd.md
cat adrs/0002-aaa-bbb.md
---
title: "2. Aaa Bbb"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0002-aaa-bbb.md
custom: foo
---

* Status: proposed
* Date: 1992-01-12

## Context and Problem Statement

cat adrs/0003-ccc-ddd.md
---
title: "3. Ccc Ddd"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0003-ccc-ddd.md
custom: foo
---

* Status: proposed
* Date: 1992-01-12

## Context and Problem Statement

